gocourse02/faker.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"github.com/brianvoe/gofakeit/v6" // fake data
```

Comment:
`gofakeit` дуже класно розкриває суть пакету, комент зайвий:
```suggestion
	"github.com/brianvoe/gofakeit/v6"
```